app/build.gradle

```diff
@@ -87,55 +66,118 @@ android {
+    implementation 'com.google.code.gson:gson:2.7' //2.8.5
+
+    /** Heterogeneous Adapter */
     implementation 'com.marverenic.heterogeneousadapter:heterogeneousadapter:1.1'
 
-    implementation "com.android.support:appcompat-v7:$rootProject.supportLibVersion"
-    implementation "com.android.support:design:$rootProject.supportLibVersion"
-    implementation "com.android.support:palette-v7:$rootProject.supportLibVersion"
-    implementation "com.android.support:recyclerview-v7:$rootProject.supportLibVersion"
-    implementation "com.android.support:cardview-v7:$rootProject.supportLibVersion"
-    implementation "com.android.support:preference-v14:$rootProject.supportLibVersion"
-    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
-    implementation 'com.google.android.exoplayer:exoplayer:2.6.1'
-
-    implementation "android.arch.persistence.room:runtime:$rootProject.roomVersion"
-    annotationProcessor "android.arch.persistence.room:compiler:$rootProject.roomVersion"
-    testImplementation "android.arch.persistence.room:testing:$rootProject.roomVersion"
+    /** RecyclerView-FastScroll */
+    // TODO: Update to latest version (https://github.com/timusus/RecyclerView-FastScroll)
+    implementation 'com.simplecityapps:recyclerview-fastscroll:1.0.16' //1.0.20
 
-    implementation 'com.github.bumptech.glide:glide:3.7.0'
-    implementation 'com.google.code.gson:gson:2.7'
+    /** Vertical SeekBar */
+    // TODO: Update to latest version (https://github.com/h6ah4i/android-verticalseekbar/blob/master/RELEASE-NOTES.md)
+    implementation 'com.h6ah4i.android.widget.verticalseekbar:verticalseekbar:0.5.2' //1.0.0
 
-    implementation 'com.simplecityapps:recyclerview-fastscroll:1.0.16'
-    implementation 'com.h6ah4i.android.widget.verticalseekbar:verticalseekbar:0.5.2'
-
-    implementation 'com.google.dagger:dagger:2.6'
-    annotationProcessor 'com.google.dagger:dagger-compiler:2.6'
-
-    implementation 'io.reactivex:rxjava:1.1.6'
-    implementation 'io.reactivex:rxandroid:1.2.1'
-    implementation 'com.trello:rxlifecycle:0.6.1'
-    implementation 'com.trello:rxlifecycle-components:0.6.1'
-    implementation 'com.tbruyelle.rxpermissions:rxpermissions:0.7.0@aar'
-    implementation 'nl.littlerobots.rxlint:rxlint:1.0'
-
-    implementation 'com.squareup.retrofit2:retrofit:2.0.2'
-    implementation 'com.squareup.retrofit2:converter-gson:2.0.2'
-    implementation 'com.squareup.retrofit2:adapter-rxjava:2.0.2'
-
-    implementation 'com.jakewharton.timber:timber:4.1.2'
-
-    implementation('com.crashlytics.sdk.android:crashlytics:2.9.1@aar') {
+    /** Crashlytics */
+    implementation('com.crashlytics.sdk.android:crashlytics:2.9.5@aar', {
         transitive = true
-    }
+    })
+
+    /** Test libraries */
+    testImplementation 'junit:junit:4.13-beta-1'
+    // TODO: Update to Robolectric 4.x (https://github.com/robolectric/robolectric/releases)
```

Comment:
We can axe this TODO as well. I'm thinking of migrating away from Robolectric once I can convince myself to rewrite some unit tests.